src/sentry/api/endpoints/sentry_app_publish_request.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry import options
+from sentry import features
+from sentry.api.bases.sentryapps import SentryAppBaseEndpoint
+from sentry.utils import email
+
+
+class SentryAppPublishRequestEndpoint(SentryAppBaseEndpoint):
+    def post(self, request, sentry_app):
```

Comment:
Related to the above comment – we should handle the case when someone makes a request to this endpoint for a Published integration. We can just bail out early with some error code and message.

Reply:
@mnoble Fixed